hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationRunnerImpl.java

```diff
@@ -89,6 +90,9 @@
     private final Address thisAddress;
     private final boolean staleReadOnMigrationEnabled;
 
+    @Probe(name = "failedBackups")
+    private final Counter failedBackupsCounter = newSwCounter();
```

Comment:
Personally I don't care of this information on a partition level, however if you want to have it, go nuts.

However the probes on non partition operationrunnerimpl should not be created. Because they will always be null and therefor cause noise.